main.go

```diff
@@ -1,9 +1,13 @@
 package main
 
 import (
+	"log"
+
 	"github.com/operator-framework/combo/cmd"
 )
 
 func main() {
-	cmd.Execute()
+	if err := cmd.Execute(); err != nil {
+		log.Fatal(err)
```

Comment:
This is purely a nit so feel free to ignore: I'm personally a fan of this error handling at the main function level:

```go
if err := cmd.Execute(); err != nil {
    fmt.Fprintln(os.Stderr, err)
    os.Exit(1)
}
```

You likely get the same functionality under-the-hood with log.Fatal so that suggestion likely falls under the personal preference realm.

Reply:
Hm, interesting. Any reason one approach is preferable to the other?